src/clientFactory.ts

```diff
@@ -25,7 +25,7 @@ export async function createClient(
     onNotification: (notification: LoggingMessageNotification) => void,
 ): Promise<Client> {
     const client = new Client(
-        { name: 'apify-mcp-client', version: '1.0.0' },
+        { name: 'apify-mcp-client', version: '1.0.1' },
```

Comment:
nit: I would do a minor bump since its a new feature.

```suggestion
        { name: 'apify-mcp-client', version: '1.1.0' },
```